package.json

```diff
@@ -16,7 +16,7 @@
   ],
   "license": "MIT",
   "private": false,
-  "version": "0.1.1",
+  "version": "0.2.0-0",
```

Comment:
typically versions are only bumped in commits direct to master, not in PRs

Reply:
is this also the case for `prerelease`?

Reply:
yes; typically you’d test things by installing fTom git, not prereleases.

All publishes need to be a git tag, prerelease or not.